unity-client/Assets/Scripts/MainScripts/DCL/Components/BuilderInWorld/DCLLockedOnEdit.cs

```diff
@@ -0,0 +1,38 @@
+using DCL;
+using DCL.Components;
+using DCL.Controllers;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// This component describes the lock status of the Entity in the builder in world.
+/// Builder in World send a message to kernel to change the value of this component in order to lock/unlock it
+/// </summary>
+public class DCLLockedOnEdit : BaseDisposable
+{
+    [System.Serializable]
+    public class Model
+    {
+        public bool value;
```

Comment:
I think this var name is a constrain of having a common pipeline, but if no... Can we rename it to something more meaningful? Something like `locked`

Reply:
The problem is that the kernel part name is "value". Since we get the model and convert it to JSON and send to kernel, we can't change the name without changing it on the kernel side

Reply:
Yeah, I assumed that. It should be quite trivial, shouldnt it?

Reply:
@nchamo what do you think?

Reply:
Why not just change the `Model` class name? that would be easier and more meaningful in my opinion... like `IsLockedModel` or something like that

Reply:
The model class is nested within `DCLLockedOnEdit` class so the full name is actually `DCLLockedOnEdit .Model` which is quite meaningful. We also have this structure across every component in our project so I wouldn't change that without changing the rest.

Reply:
Renamed to `isLocked` @nchamo  let me know when you change the name so I can push the changes!

Reply:
You can go ahead with the changes, there is nothing to be done on Kernel's side. Same with the other component